README.md

```diff
@@ -87,15 +87,15 @@ or
 ```
 
 - Pulling your request. Click on the Pull requests tab on the forked github repository.
-  Note : A pull request allows your changes to be merged with the original project.
+  Note: A pull request allows your changes to be merged with the original project.
 
 ```
   click on pull request
```

Comment:
I guess in the new update its changed to `Contribute > Pull Request` could please add that step here too? Coz that's creating a bit of confusion too.

Reply:
@schmelto as you have resolved https://github.com/EddieHubCommunity/hacktoberfest-practice/pull/368#discussion_r659270194 you can mark this as resolved too